src/Fixer/ClassNotation/StaticPrivateMethodFixer.php

```diff
@@ -0,0 +1,298 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\ClassNotation;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+use PhpCsFixer\Tokenizer\TokensAnalyzer;
+
+/**
+ * @author Filippo Tessarotto <zoeslam@gmail.com>
+ */
+final class StaticPrivateMethodFixer extends AbstractFixer
+{
+    /**
+     * @var array
+     */
+    private $magicNames = [
```

Comment:
Can it be constant?

Reply:
Yes, must it?

Reply:
I'd say yes, as for [POLA](https://en.wikipedia.org/wiki/Principle_of_least_astonishment).

Always when I see variable I'd expect it would be modified and when I see constant I know it won't be modified - and here we have variable that won't be modified.

Reply:
Of course. What stops me from doing this is that here we must use public constants as this project must support PHP versions that don't know about private constants.

Between public constants and private variables, I'd prefer private variables, WDYT?

Reply:
> Between public constants and private variables

we can mark constant `@internal`, it make a bit ~better~ less bad, but I won't insist.

Reply:
This discussion is outdated nowadyas, we can use `private const` now.